src/components/YourCompany/CorrectBusinessType.vue

```diff
@@ -283,6 +306,13 @@ export default class CorrectBusinessType extends Mixins(CommonMixin) {
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
 
+.help-section {
+  .subtitle, .info-text {
+    font-size: .875rem !important;
+    line-height: 1.375rem !important;
```

Comment:
It would be nice to use existing classes, if they exist (and are suitably declared).

Reply:
Done and done, this is gone in next commit.